packages/react-native-windows-init/src/Cli.ts

```diff
@@ -575,25 +607,25 @@ function isProjectUsingYarn(cwd: string): boolean {
       }
     }
 
-    installReactNativeWindows(version, useDevMode);
+    installReactNativeWindows(version, !!options.verbose, useDevMode);
 
     const generateWindows = requireGenerateWindows();
 
     // Now that new NPM packages have been installed, refresh their versions
     await Telemetry.populateNpmPackageVersions(true);
 
     await generateWindows(process.cwd(), name, ns, {
-      language: argv.language as 'cs' | 'cpp',
-      overwrite: argv.overwrite,
-      verbose: argv.verbose,
-      projectType: argv.projectType as 'lib' | 'app',
-      experimentalNuGetDependency: argv.experimentalNuGetDependency,
-      useWinUI3: argv.useWinUI3,
-      useHermes: argv.useHermes,
+      language: options.language as 'cs' | 'cpp',
+      overwrite: !!options.overwrite,
```

Comment:
The type of `options` looks to be less precise than that of `argv`, which knew this was a never-nullish boolean. `yargs` will infer the type of options based on the values passed into its options params.